aio_pika/queue.py

```diff
@@ -63,7 +66,10 @@ def declare(self, timeout: int = None, passive: bool = False) -> asyncio.Future:
         )
 
         def on_queue_declared(result):
-            self.name = result.result().method.queue
+            method = result.result().method
+            self.name = method.queue
+            self.message_count = method.message_count
+            self.consumer_count = method.consumer_count
```

Comment:
Actually, given the attempt of the library to have a `Queue` abstraction I think storing them on the `Queue` makes the most sense (especially since there's no way of creating a `Queue` except by `channel.declare_queue`). They could perhaps be renamed to `last_message_count`/`last_consumer_count` or something, to make it clear that the values may be out of date.